sql/pre_install/tables.sql

```diff
@@ -358,6 +358,8 @@ CREATE TABLE IF NOT EXISTS _timescaledb_catalog.compression_chunk_size (
     compressed_heap_size BIGINT NOT NULL,
     compressed_toast_size BIGINT NOT NULL,
     compressed_index_size BIGINT NOT NULL,
+    uncompressed_total_rows BIGINT ,
+    compressed_total_rows BIGINT ,
```

Comment:
```suggestion
    uncompressed_total_rows BIGINT,
    compressed_total_rows BIGINT,
```

Reply:
I wish there was a better name for these columns, but I certainly haven't thought of one...

Reply:
the thing that confuses me is that it’s not obvious that the `uncomprssed_rows` are in fact all compressed now, and it’ll be even more confusing if we allow partially-compressed chunks. Maybe `rows_pre_compression` works better?

Reply:
renamed vars.